cfgov/unprocessed/apps/regulations3k/css/reg-navigation.less

```diff
@@ -35,7 +35,7 @@
         &-left {
             width: 88%;
         }
-        &-left {
+        &-right {
```

Comment:
This appears to have been a typo in https://github.com/cfpb/consumerfinance.gov/commit/e1ddf9d8ec2d4193b6de42fef4337d5cb2507d40 when migrating 
<img width="345" alt="Screen Shot 2022-04-15 at 1 22 43 PM" src="https://user-images.githubusercontent.com/704760/163602682-fc96dd25-9473-47cf-b38e-7be3245c9bb9.png">